test/publicpathoption.test.js

```diff
@@ -0,0 +1,30 @@
+const test = require('ava');
+const path = require('path');
+const fs = require('fs-extra');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const WebappWebpackPlugin = require('../');
+
+const { logo, generate, mkdir, compare, expected } = require('./util');
+
+test.beforeEach(async t => t.context.root = await mkdir());
+
+test('should allow for overriding the publicPath option', async t => {
+  const dist = path.join(t.context.root, 'dist');
+
+  console.log(dist);
+  await generate({
+    context: t.context.root,
+    output: {
+      path: dist,
+      publicPath: '/public/path',
+    },
+    plugins: [
+      new HtmlWebpackPlugin(),
+      new WebappWebpackPlugin({ logo, publicPath: '/public/path/deeper' }),
```

Comment:
Would you mind changing this to a something unrelated to the actual `publicPath`?

Reply:
Do you mean making it something like `/another/path/excluding/public`?

Reply:
Yeah or `a/b/`, I mean anything that doesn't suggest it has to relate to the webpack `publicPath`